charts/druid/values.yaml

```diff
@@ -0,0 +1,4 @@
+image: eu.gcr.io/gardener-project/gardener/etcd-druid
+version: v0.6.0-dev-a47e74fd1756951720f8f309cfa34060c677c021
```

Comment:
Can you remove this default value? I don't think we should have a dev version in Helm chart we offer.